exec/fixture/infinite.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
this won't work on non-unix platforms. could be addressed in a follow-up PR if needed